### PR TITLE
Feature: Alerts with layer data

### PR DESCRIPTION
**Pull request by migh**

### CODE REVIEW

File: src/components/widget/templates/mangrove-alerts/component.js [hunk 1]
```diff
@@ -49,11 +49,57 @@ function MangroveAlerts({ widgetConfig, minDate = '1996-01-01', maxDate = '2016-
...
 
-  const { chartData, chartConfig } = widgetConfig.parse({ startYear, endYear });
-  const alerts = chartData.map(it => it.year).filter(y => y >= startYear && y <= endYear ).length;
+  const seriesLineDefinition = {
+    stroke: 'red',
+    strokeWidth: 1,
+    isAnimationActive: false,
+    dot: {
+      fill: 'red',
+      stroke: 'red',
```
Comment (mluena) [inline src/components/widget/templates/mangrove-alerts/component.js]: According to design this color should be #EB6240
```suggestion
      stroke: '#EB6240',
```

File: src/components/widget/templates/mangrove-alerts/component.js [hunk 2]
```diff
@@ -49,11 +49,57 @@ function MangroveAlerts({ widgetConfig, minDate = '1996-01-01', maxDate = '2016-
...
+  const endMark = month(state.endDate) - 2;
 
-  const { chartData, chartConfig } = widgetConfig.parse({ startYear, endYear });
-  const alerts = chartData.map(it => it.year).filter(y => y >= startYear && y <= endYear ).length;
+  const seriesLineDefinition = {
+    stroke: 'red',
+    strokeWidth: 1,
+    isAnimationActive: false,
+    dot: {
+      fill: 'red',
```
Comment (migh) [inline src/components/widget/templates/mangrove-alerts/component.js]:
```suggestion
      fill: '#EB6240',
```

File: src/components/widget/templates/mangrove-alerts/component.js (hunk 1 above)
Comment (migh) [inline src/components/widget/templates/mangrove-alerts/component.js]:
```suggestion
      stroke: '#EB6240',
```

File: src/components/widget/templates/mangrove-alerts/component.js (hunk 2 above)
Comment (mluena) [inline src/components/widget/templates/mangrove-alerts/component.js]:
```suggestion
      fill: '#EB6240',
```

File: src/components/widget/templates/mangrove-alerts/component.js [hunk 3]
```diff
@@ -49,11 +49,57 @@ function MangroveAlerts({ widgetConfig, minDate = '1996-01-01', maxDate = '2016-
...
 
-  const startYear = year(state.startDate);
-  const endYear = year(state.endDate);
+  const startMark = month(state.startDate) + 2;
+  const endMark = month(state.endDate) - 2;
 
-  const { chartData, chartConfig } = widgetConfig.parse({ startYear, endYear });
-  const alerts = chartData.map(it => it.year).filter(y => y >= startYear && y <= endYear ).length;
+  const seriesLineDefinition = {
+    stroke: 'red',
```
Comment (migh) [inline src/components/widget/templates/mangrove-alerts/component.js]:
```suggestion
    stroke: '#EB6240',
```

**Pull request merged**